### PR TITLE
add theme switch button to footer

### PR DESCRIPTION
**Pull request by manuelwecker**
Bildschirmfoto 2020-01-09 um 19 23 30
Bildschirmfoto 2020-01-09 um 19 23 22

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -26,8 +27,22 @@ const Container = styled.div`
 `;
 
 function App() {
+  const [activeTheme, setActiveTheme] = React.useState(light);
+
+  function onClick() {
+    // setActiveTheme({props => props.theme.dark});
```
Comment (holgermanzke) [inline client/src/App.js]: remove comments ;)

File: client/src/App.js [hunk 2]
```diff
@@ -26,8 +27,22 @@ const Container = styled.div`
...
+    // setActiveTheme(dark);
+
+    if (activeTheme === light) {
+      setActiveTheme(dark);
+    } else {
+      setActiveTheme(light);
+    }
+    // console.log(activeTheme);
```
Comment (holgermanzke) [inline client/src/App.js]: no comments & no console.log

File: client/src/App.js (hunk 1 above)
Comment (manuelwecker) [reply client/src/App.js]: thx

File: client/src/App.js (hunk 2 above)
Comment (manuelwecker) [reply client/src/App.js]: done

File: client/src/components/Header.js [hunk 3]
```diff
@@ -18,8 +18,8 @@ const HeaderNavigation = styled.div`
 
 const ButtonNavigation = styled.button`
   color: #ffffff;
-  border: 1px solid #ffffff;
-  border-radius: 4px;
+  border: 0px solid #ffffff;
```
Comment (panchepanevski) [inline client/src/components/Header.js]: I would pack the `#ffffff` color in Theme.

File: client/src/pages/IssueList.js [hunk 4]
```diff
@@ -18,28 +21,65 @@ export default function IssueList() {
...
+  const Span = styled.span`
+    text-align: middle;
+    &:a {
+      text-decoration: none;
+    }
+  `;
+  const TestButton = styled.button`
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (panchepanevski) [inline client/src/pages/IssueList.js]: 2 x `justify-content` I'm not sure did they work properly.

**Pull request merged**